src/sys.rs

```diff
@@ -926,7 +922,8 @@ impl SysOp {
                 env.push(vec![height as f64, width as f64])
             }
             SysOp::Args => {
-                let args = env.backend.args();
+                let mut args = env.args();
+                args.extend(env.args());
```

Comment:
Why are the args put in the list twice?

Reply:
Whoops, that should've been the file path, of course.